app/views/spree/users/_fat.html.haml

```diff
@@ -16,6 +16,8 @@
           %td.order3.show-for-large-up
             %i{"ng-class" => "{'ofn-i_012-warning': payment.state == 'invalid' || payment.state == 'void' || payment.state == 'failed'}"}
             %span{"ng-bind" => "::'spree.payment_states.' + payment.state | t | capitalize"}
+            %span{"ng-if" => "payment.cvv_response_message.length > 0" }
+              %a{"ng-href" => "{{payment.cvv_response_message}}", "ng-bind" => "::'spree.payment_states.authorise' | t | capitalize" }
```

Comment:
There's something a bit confusing from the resulting UI (notice I faked the payment a bit. I just provided a `cvv_response_message` on an existing payment). On that page, we highlight the balance in red but it also indicates a link.
![Screenshot from 2021-02-10 17-07-55](https://user-images.githubusercontent.com/762088/107536888-95717400-6bc2-11eb-8eb6-969107bc9552.png)

Wouldn't it be more understandable for the user to highlight the required action a bit more somehow? What are your thoughts on this @Erioldoesdesign ?  I wonder if users will notice they need to click there... we check that in production though and improve it in another PR.

Reply:
Talked offline with  @andrewpbrett and this is just a fallback from the email notification users get. If they lose the email they can come here and still authorize the payment.

Reply:
I agree with you @sauloperez: this is not obvious that this link is clickable. And its more a CTA than a simple link (like `Order R25...`). Maybe we should have _small button components_ to meet this need.

Reply:
yes, I feel we need to explore components that are neither CTAs nor plain old links :ok_hand: I hope this fits into feature discussions about a style guide.

Reply:
I made them buttons for now: #6919